src/summary.ts

```diff
@@ -0,0 +1,163 @@
+import * as core from '@actions/core'
+import {ConfigurationOptions, Change, Changes} from './schemas'
+import {SummaryTableRow} from '@actions/core/lib/summary'
+
+export function addSummaryToSummary(
+  addedPackages: Changes,
+  licenseErrors: Change[],
+  unknownLicenses: Change[]
+): void {
+  core.summary
+    .addHeading('Dependency Review')
+    .addRaw(
+      `We found ${addedPackages.length} vulnerable package(s), ${licenseErrors.length} package()s with incompatible licenses, and ${unknownLicenses.length} package(s) with unknown licenses.`
```

Comment:
Looks like there's a typo in "package()s":

```suggestion
      `We found ${addedPackages.length} vulnerable package(s), ${licenseErrors.length} package(s) with incompatible licenses, and ${unknownLicenses.length} package(s) with unknown licenses.`
```

Reply:
Good catch.

Thank you